### DISCUSSION TITLE
Just wondering if MTEX is capable of handling or improt  cpr file?

### DISCUSSION THREAD

**Discussion by rpshi** [mtex-toolbox/mtex#1613] [Ask Anything]
On the webpage https://mtex-toolbox.github.io/EBSDExport.html, it says that "
MTEX allows you to export EBSD data as [h5](https://mtex-toolbox.github.io/EBSD.export_h5.html), [ctf](https://mtex-toolbox.github.io/EBSD.export_ctf.html), [ang](https://mtex-toolbox.github.io/EBSD.export_ang.html) or [cpr/crc](https://mtex-toolbox.github.io/EBSD.export_cpr.html) files."

On the webpage, https://mtex-toolbox.github.io/EBSDImport.html, under the section of "Supported Data Formats", I saw crc but not cpr.

So, Just wondering if MTEX is able to import and analyze cpr files.

Thanks for your time.

Best regards,

R.

**Comment by kilir** [comment]
Hi,
those two file types belong together.
Cheers,
Rüdiger

**Comment by rpshi** [comment]
thanks, but it seems not the import function won't allow for importing cpr/crc files?

**Comment by zmichels** [comment]
It does work — I’ve never had a problem. You have to import the CRC file, and you may-or-may-not need the .cpr file in the same directory (I’ve never needed it, but some folks tell me they do). The CRC file contains all the data… the CPR file contains “project” information… the two together were used by Oxford Instruments to define datasets within projects. However… the data you want to import are in the CRC file. As far as I can tell, the CPR files are useless for data processing without the paired CRC file, and for MTEX sake, you want to setup your import to target the CRC file.Cheers,ZachOn Oct 30, 2022, at 7:55 PM, rpshi ***@***.***> wrote:﻿
thanks, but it seems not the import function won't allow for importing cpr/crc files?

**Comment by rpshi** [comment]
Yes, it works. I just try importing a crc file using Import wizard.

Thanks
